person/models.py

```diff
@@ -100,7 +100,7 @@ class Person(models.Model):
     surname = models.CharField(
```

Comment:
I don't know if first and last names should also be null and blank true. Several members would say why not, if they don't want to put it, don't put it. As far as I'm concerned, it's a bit the same. Another change can be to put surname in the plural, for the two surnames, and therefore also change the verbose name. I'm going to ask you for changes, but depending on whether you see it logical or not, you implement it and let me know.